### ISSUE TITLE
A11y: File/folder list is missing UIA names, faulty controltype & localizedcontroltype 

### ISSUE THREAD

**Issue by chingucoding** [files-community/Files#2570]
**Describe the bug**
The default file/folder list is missing UIA names for multiple things:
1. The image in the first column
2. The second hidden image which get's reported by accessibility insights and narrator
3. The value for the "Size" column if there is no size present

In addition to that, the first column of the file/folder view is using the "Custom" controltype and is not providing a localized control type, resulting in users having no idea what the element actually is.

Also, the status column is also affected by this:
1. Control with controltype "custom" and no localized control type

Here are screenshots to clarify what I mean:

![Folder view with accessibility issues highlighted](https://user-images.githubusercontent.com/16122379/101672035-a0d6e080-3a55-11eb-9f24-39f1011d59b8.png)

![Image showing the control tree of the accessibility issues in question](https://user-images.githubusercontent.com/16122379/101672107-bcda8200-3a55-11eb-8998-a926622b765d.png)

![Image of folder with status column issue pointed out](https://user-images.githubusercontent.com/16122379/101676603-aa634700-3a5b-11eb-932b-a6800f88abae.png)

**To Reproduce**
1. Open Files
2. Open a folder in Files
3. Use narrator item mode to narrate the entire line of an item
4. Observe the narrated content

**Expected behavior**
In the case of the size column, it might make sense to announce something like "not calculated" instead of nothing.
In the case of the custom controltype without localized controltype. either use non custom controltype, provide a localized contorltype or even remove it from the control tree if it does not add information to users.

Not sure why there are two images in the control tree, this seems like a bug. Maybe remove them from the control tree too if it makes no sense do discern the first column as multiple parts?

**Desktop (please complete the following information):**
- OS: Insider Build 10.0.20262 Build 20262
- App version: 0.21.1.0

**Additional context**
Add any other context about the problem here. For instance, are you building from source or using a precompiled build/snapshot?

**Log file**
Please post the log file here so that we can understand your problem better. You can access it from Settings->About->Open log location.
[debug.txt](https://github.com/files-community/Files/files/5667956/debug.txt)
[debug_fulltrust.txt](https://github.com/files-community/Files/files/5667957/debug_fulltrust.txt)

**Comment by yaichenbaum** [comment]
@winston-de Can you confirm if this is resolved in `main`?

**Comment by winston-de** [comment]
@yair100 It is not, but it's something I'm working on for the new details view. Previously this was blocked by https://github.com/windows-toolkit/WindowsCommunityToolkit/issues/3400.

**Comment by yaichenbaum** [comment]
@chingucoding This issue should be resolved in `main`

**Comment by chingucoding** [comment]
Awesome, thank you for fixing this!
